### PR TITLE
Update Troubleshooting slides

### PR DESCRIPTION
**Pull request by natefoo**
Still need to work out how each of the demos will go and how I can capture those into the training material later, but hey I still have hour(s) left.

### CODE REVIEW

File: topics/admin/tutorials/troubleshooting/slides.html [hunk 1]
```diff
@@ -6,64 +6,442 @@
...
+
+## LOGS
+
+--
+
+.left[Check **all** the logs]
+- uWSGI log
+- handler logs (if not all-in-one)
+- Pulsar logs
+- nginx access, error logs
```
Comment (nsoranzo) [inline topics/admin/tutorials/troubleshooting/slides.html]:
```suggestion
- nginx error and access logs
```

File: topics/admin/tutorials/troubleshooting/slides.html [hunk 2]
```diff
@@ -6,64 +6,442 @@
...
+# Most common problems
+
+- Startup problems
+- Web/UI problems
+- Tool failures
+- Job execution problems
+- General performance problems
+- Other stuff
+- ...
+- ...?ASF?ASD?FAS?DF
```
Comment (nsoranzo) [inline topics/admin/tutorials/troubleshooting/slides.html]: ?

File: topics/admin/tutorials/troubleshooting/slides.html [hunk 3]
```diff
@@ -6,64 +6,442 @@
...
+- Clean the `*.pyc` files in migrate versions folder to make sure there is no remnant from other code revisions.
+  - See [makepyc.py in galaxyproject.galaxy](https://github.com/galaxyproject/ansible-galaxy/blob/master/files/makepyc.py).
+
+---
+class: left
 # Database migration
 
-In case you see database migration errors during startup you can:
+**Downgrading** - Perform in this order:
+  1. Downgrade the DB
```
Comment (nsoranzo) [inline topics/admin/tutorials/troubleshooting/slides.html]:
```suggestion
  1. Downgrade the DB with `manage_db.sh`
```

File: topics/admin/tutorials/troubleshooting/slides.html [hunk 4]
```diff
@@ -6,64 +6,442 @@
...
+  2. Downgrade Galaxy
```
Comment (nsoranzo) [inline topics/admin/tutorials/troubleshooting/slides.html]:
```suggestion
  2. Downgrade Galaxy with `git checkout`
```

File: topics/admin/tutorials/troubleshooting/slides.html [hunk 5]
```diff
@@ -6,64 +6,442 @@
...
+# Most common problems
+
+- Startup problems
+- Web/UI problems
+- Tool failures
+- Job execution problems
+- General performance problems
+- Other stuff
+- ...
+- ...?ASF?ASD?FAS?DF
```
Comment (natefoo) [inline topics/admin/tutorials/troubleshooting/slides.html]:
```suggestion
```

**Pull request merged**